mods/ra/rules/structures.yaml

```diff
@@ -743,7 +743,7 @@ HBOX:
 		Range: 5c0
 	Cloak:
 		InitialDelay: 125
-		CloakDelay: 60
+		CloakDelay: 125
```

Comment:
I'd say either price hike or cloak delay increase, but not both at the same time. That sound's like too much of a nerf.

Reply:
I agree, especially considering the cloak targeting fix that removes their unwanted untargetable-under-fog buff.